python/pypto/pypto_core/__init__.py

```
@@ -0,0 +1,77 @@
+    for path in _iter_extension_candidates():
+        loader = importlib.machinery.ExtensionFileLoader(__name__, str(path))
+        spec = importlib.util.spec_from_file_location(__name__, path, loader=loader)
+        if spec is None or spec.loader is None:
+            errors.append(f"{path}: failed to create import spec")
+            continue
+        module = importlib.util.module_from_spec(spec)
+        sys.modules[__name__] = module
+        try:
+            spec.loader.exec_module(module)
+            return module
+        except Exception as exc:  # pragma: no cover - exercised only on broken builds
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)